### ISSUE TITLE
SonarLint: proversion has no value in preprocess condition

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1206]
In both PDSOE and VS Code the ``proversion`` expression seems to have no value when used in a preprocess condition.

````
&if proversion >= '12.8' &then

   VAR INT foo. // should be flagged unused (based on current runtime)

&else

   DEF VAR foo AS INT NO-UNDO. // flagged as unused, so &else branch taken

&endif
````
